### DISCUSSION TITLE
JetBrains IDEs Private Beta access granted but unavailable in settings of account

### DISCUSSION THREAD

**Discussion by StephenPearsonDev** [community/community#101821] [Copilot Conversations]
### Select Topic Area

Question

### Body

Hi,

A couple of days ago I received the email granting me access to the copilot private beta. I have tried following the instructions to modify the policies but chat is not visible on my account (https://docs.github.com/en/copilot/github-copilot-chat/using-github-copilot-chat-in-your-ide?tool=jetbrains). 

I see the regular copilot settings - do I need to do something else to enable chat? Will this not be available for my account or does it take time?

Thanks

**Comment by Akash1134** [comment]
Hi @StephenPearsonDev ,

Have you signed in to Copilot plugin using your JetBrains IDE? You need to authenticate the device code (see [instructions](https://docs.github.com/en/copilot/github-copilot-chat/using-github-copilot-chat-in-your-ide?tool=jetbrains_beta#:~:text=In%20the%20%22Sign%20in%20to%20GitHub%22%20dialog%20box%2C%20to%20copy%20the%20device%20code%20and%20open%20the%20device%20activation%20window%2C%20click%20Copy%20and%20Open.))

Also, are you using the latest version of both Copilot (>= 1.4.0 is required) and JetBrains IDE? 

Thanks!

**Comment by StephenPearsonDev** [comment]
Hi @Akash1134 @DarrenChochian Thank you for your input. I did a clean install of intellij and got everything up to date. The chat box is functioning now, so something must have been out of date. Interestingly, on my github settings page for copilot it has no mention of  the chat permissions, so maybe the docs for this need to be modified.
